### PR TITLE
New package: uucp-1.07

### PR DESCRIPTION
**Pull request by bugcrazy**
This package contains the standard UUCP package from the Free Software Foundation, with uucico, uusched, uuxqt, uux, uucp, uustat, uulog, uuname, uuto, uupick and cu.

### CODE REVIEW

**Comment by xtraeme** [comment]
Uudontknow

File: srcpkgs/uucp/INSTALL [hunk 1]
```diff
@@ -0,0 +1,7 @@
+case ${ACTION} in
+post)
+	# fix permissions and owners
+	chown -R uucp:uucp usr/bin/uucico usr/bin/uuxqt usr/bin/uux usr/bin/uucp usr/bin/uustat usr/bin/uuname	usr/bin/cu
```
Comment (leahneukirchen) [inline srcpkgs/uucp/INSTALL]: -R is useless here

**Comment by bugcrazy** [comment]
> Uudontknow

It is a free software implementation of the Unix UUCP package.

File: srcpkgs/uucp/patches/uucp-org.patch [hunk 2]
```diff
@@ -0,0 +1,8833 @@
...
+-Taylor UUCP version of `uux'.  If you only have Taylor UUCP, or if you
+-simply replace any existing version of `uux' with the Taylor UUCP
+-version, this will probably happen automatically.  However, if you have
+-two UUCP packages installed on your system, you will probably have to
+-modify the mail and news configuration files in some way.
+-
+-   Actually, if both the system UUCP and Taylor UUCP are using the same
+-spool directory format, the system `uux' will probably work fine with
+-the Taylor `uucico' (the reverse is not the case: the Taylor `uux'
+-requires th
```
Comment (Chocimier) [inline srcpkgs/uucp/patches/uucp-org.patch]: 8 thousand line patch? Coudn't debian make new release?

File: srcpkgs/uucp/template [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Template file for 'uucp'
+pkgname=uucp
+version=1.07
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man --with-newconfigdir=/etc/uucp --sbindir=/usr/bin"
```
Comment (Chocimier) [inline srcpkgs/uucp/template]: all but newconfigdir are part of build-style

File: srcpkgs/uucp/template [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+makedepends="wget"
+short_desc="Taylor UUCP is a free implementation of UUCP"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://www.gnu.org/software/uucp/uucp.html"
+distfiles="${GNU_SITE}/$pkgname/$pkgname-$version.tar.gz"
+checksum=060c15bfba6cfd1171ad81f782789032113e199a5aded8f8e0c1c5bd1385b62c
+patch_args="-Np1"
+
+system_accounts="uucp"
```
Comment (Chocimier) [inline srcpkgs/uucp/template]: Should be prefixed with underscore per Manual.md.

File: srcpkgs/uucp/template [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+uucp_homedir="/var/spool/uucp"
+
+make_dirs="
+	/var/spool/uucp		0750 uucp uucp
+	/var/spool/uucppublic	0750 uucp uucp"
+
+pre_configure() {
+	sed -i 's|${CC-cc} -E|${CC-cc} -E -O2|g' configure
```
Comment (Chocimier) [inline srcpkgs/uucp/template]: Why is -O2 important for preprocesing?

File: srcpkgs/uucp/template [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+}
+do_install() {
+	make DESTDIR=${DESTDIR} install
+	vmkdir etc/uucp/
+	vinstall sample/config 0644 etc/uucp/
```
Comment (Chocimier) [inline srcpkgs/uucp/template]: They should be added to `conf_files`.

File: srcpkgs/uucp/patches/uucp-org.patch (hunk 2 above)
Comment (bugcrazy) [reply srcpkgs/uucp/patches/uucp-org.patch]: It seems that the original author did not fix simple errors, Debian only made this big patch, to fix trivial things. I agree should have released a version.

File: srcpkgs/uucp/template (hunk 3 above)
Comment (bugcrazy) [reply srcpkgs/uucp/template]: The original folder is /usr/conf/uucp, the most logical path is /etc/uucp, as they are configuration files.

File: srcpkgs/uucp/template (hunk 5 above)
Comment (bugcrazy) [reply srcpkgs/uucp/template]: It is used for optimization.

File: srcpkgs/uucp/template (hunk 6 above)
Comment (bugcrazy) [reply srcpkgs/uucp/template]: Done.

File: srcpkgs/uucp/template (hunk 4 above)
Comment (bugcrazy) [reply srcpkgs/uucp/template]: Done.

File: srcpkgs/uucp/template (hunk 3 above)
Comment (Chocimier) [reply srcpkgs/uucp/template]: I meant --prefix and others are passed due to build-style anyway, no need to define them here.

File: srcpkgs/uucp/template (hunk 4 above)
Comment (Chocimier) [reply srcpkgs/uucp/template]: Uses of the account name should be changed too.

File: srcpkgs/uucp/template (hunk 5 above)
Comment (Chocimier) [reply srcpkgs/uucp/template]: -E means textual inclusion of headers and expanding macros. There are no optimizations to apply.

File: srcpkgs/uucp/template (hunk 6 above)
Comment (Chocimier) [reply srcpkgs/uucp/template]: 👌

File: srcpkgs/uucp/template (hunk 3 above)
Comment (bugcrazy) [reply srcpkgs/uucp/template]: Done.

File: srcpkgs/uucp/template (hunk 4 above)
Comment (bugcrazy) [reply srcpkgs/uucp/template]: Done.

File: srcpkgs/uucp/template (hunk 5 above)
Comment (bugcrazy) [reply srcpkgs/uucp/template]: Removed -02.

**Comment by Anachron** [comment]
Please squash commits.

**Comment by bugcrazy** [comment]
> Please squash commits.

Completed the squash!

**Comment by bugcrazy** [comment]
PR is waiting for approval or not!?

**Comment by leahneukirchen** [comment]
If you are still interested in having this, please rebase to master for CI to run.

**Comment by bugcrazy** [comment]
> If you are still interested in having this, please rebase to master for CI to run.

Thanks for answering!

**Pull request merged**